cloud-functions/src/utlils/getTransactionsByAddress.ts

```diff
@@ -49,7 +49,11 @@ export default async function getTransactionsByAddress(
 					amount_token: String(transfer.amount),
 					block_number: Number(transfer.block_num),
 					network: network,
-					note: transactionDoc.exists && storedTransaction?.note ? storedTransaction?.note : ''
```

Comment:
@alphainfinitus I think we need to add notifications on queues not in transactions `src\utlils\getMultisigQueueByAddress.ts`